src/lib/y2firewall/dialogs/change_zone.rb

```diff
@@ -0,0 +1,122 @@
+# encoding: utf-8
+
+# Copyright (c) [2018] SUSE LLC
+#
+# All Rights Reserved.
+#
+# This program is free software; you can redistribute it and/or modify it
+# under the terms of version 2 of the GNU General Public License as published
+# by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+# FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
+# more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, contact SUSE LLC.
+#
+# To contact SUSE LLC about this file by physical or electronic mail, you may
+# find current contact information at www.suse.com.
+
+require "cwm/dialog"
+require "y2firewall/widgets/zone_options"
+
+module Y2Firewall
+  module Dialogs
+    # This dialog allows the user to select which zone should be an interface assigned to.
```

Comment:
So, do you prefer a normal dialog instead of a Popup one? IMHO it looks like too much space for just a combo box but feel free to ignore this comment.